content/blog/managing-aws-credentials-on-cicd/index.md

```diff
@@ -0,0 +1,231 @@
+        Statement: [{
+            Action: [ "ec2:Describe*" ],
+            Effect: "Allow",
+            Resource: "*"
+        }]
+    })
+});
+
+const policy = new aws.iam.Policy("mypolicy", {
+    policy: JSON.stringify({
+        Version: "2012-10-17",
+        Statement: [{
+            Action: [
+              "ec2:Describe*"
+            ],
+            Effect: "Allow",
+            Resource: "*"
+        }]
+    })
+});
+
+const rolePolicyAttachment = new aws.iam.RolePolicyAttachment("myrolepolicyattachment", {
+    role: role,
+    policyArn: policy.arn
+});
+
+const user = new aws.iam.User("myuser");
+
+const group = new aws.iam.Group("mygroup");
+
+const policyAttachment = new aws.iam.PolicyAttachment("mypolicyattachment", {
+    users: [user],
+    groups: [group],
+    roles: [role],
+    policyArn: policy.arn
+});
+```
+
+### Isolating with Different Accounts
+
+A common and **recommended** practice is to use multiple AWS accounts to isolate resources and users from one another. For example, if your testing and production environments are in different AWS accounts, it would be difficult for a severe bug in the testing environment to harm your customer data.
+
+Keep this in mind as we configure this IAM User and later grant them IAM Roles to perform cloud deployments. Creating this specific IAM User to CI/CD jobs with its AWS account means that there are no resources to modify, in addition to not having permissions to do anything.
+
+## Provide that IAM user’s credentials to your CI/CD system
+
+Now that we have created an IAM user account, we need to give those credentials to the CI/CD system so they can be used to make changes to infrastructure as needed.
+
+We need to take great caution here. In fact, [AWS’s documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) states, **“Do not provide your access keys to a third party.”**
+
+Providing AWS access keys to any system is dangerous because of the risk that a bad actor could acquire those keys to do something nefarious. Even if you do trust your CI/CD service provider to store these credentials properly, the keys could be inadvertently exposed in debugging output, system logs, or some other place they could be disclosed.
+
+However, if you want to automate changes to your AWS infrastructure from a 3rd party CI/CD service, you must provide those credentials. Here is where following security best practices come into play.
+
+If we do need to provide credentials to a 3rd party, rather than hoping it will be 100% secure forever (which is impossible), we can instead make those credentials volatile. If we regularly invalidate and rotate the credentials given supplied to CI/CD system, we can dramatically reduce the impact of any accidental disclosure. Since even if the IAM User’s credentials were leaked, by the time they were discovered and used, they would no longer be valid. We’ll go into how to manage key rotation shortly.
```

Comment:
Just to tie this section to the overall theme of "this is how you setup using Pulumi to deploy apps on AWS via CI/CD", you might want a final paragraph making it super-clear what the action is:

> So the next step is to take the credentials for the IAM user we created earlier, and enter those into your CI/CD system so that they are available during the next build.